### DISCUSSION TITLE
SQLAlchemy admin panel support

### DISCUSSION THREAD

**Discussion by FloareDor** [benavlabs/FastAPI-boilerplate#150] [Ideas]
It would also be helpful and useful to have an admin panel built-in.

like anyone one of the following:

https://github.com/amisadmin/fastapi-amis-admin
https://github.com/aminalaee/sqladmin
https://github.com/jowilf/starlette-admin

thank you so much for FastCRUD and this boilerplate in the first place. you're a legend.

**Comment by igorbenav** [comment]
Hey, @FloareDor, I'm glad you like it!

I was actually creating an admin dashboard with FastCRUD and HTMX, but noticed it will take a lot more time (since I still have to do other stuff), and kind of abandoned this issue.

I think adding sqlalchemy admin is good, I'll do it eventually, but you may do it if you want to.
